python/py-jsonnet/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        google jsonnet 0.14.0 v
+revision            0
+name                py-${github.project}
+categories-append   devel
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         Python wrapper for ${github.project}, the data templating language
+
+long_description    A data templating language for app and tool developers
+
+homepage            http://jsonnet.org/
+
+checksums           rmd160  2e0b39c5070fc02c9b645e2e638a516d0effe44a \
+                    sha256  3521b8fa65c4cf06c1ff7daba5792304fadc87aa640cbdcb613fbe2fecaed0fb \
+                    size    21862029
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_lib-append \
+                    port:jsonnet
+
+    compiler.cxx_standard 2011
+
+    pre-build {
+        system -W ${worksrcpath} "make"
+    }
```

Comment:
I'm not understanding why you are doing this? This is the default build phase command.

Reply:
Is this because the python portgroup changes the build phase? If so, just set the `build.cmd` manually instead.

`build.cmd make
`

Reply:
That will break the Python build.

For some reason, setup.py wants to see all the build binaries from make here.

I don’t know encounters about setup.py to explain why. But I know that make then running setup.py as a standard MacPorts python port works.

Reply:
See https://github.com/macports/macports-ports/blob/46fc86b7595cd4ca581efa7dce6ac3962f005ac6/_resources/port1.0/group/python-1.0.tcl#L272 for why you can't edit `build.cmd`.

The idea is to run `make`, then let the python build take over.